src/components/Results/Credibility.js

```diff
@@ -56,6 +56,22 @@ const useStyles = makeStyles({
 
 function Credibility(props) {
   const classes = useStyles();
+  let score;
+  let category;
+  let source;
+
+  const handleError = () =>{
+    if (isNaN(props.score)){
```

Comment:
I think @ANA-POTJE could chime in here on how errors are 'created'?

I think the best way to detect an error is to test if results[0].outcome.error is defined - if it is, it means an error occurred... otherwise outcome.score, outcome.category etc should all be set.

Reply:
If the news website has no rating (nothing is returned from the Gate Source API), then we will get the following message from lambda: "error" : "The credibility score was not available."
E.g. please check the URL = https://socialistworker.co.uk/

Reply:
I will have a look, thanks guys.